ICM: Failing nfs-persistence

```
errorOutput: {}+ ./helm_install.sh
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /data/k8s/kube-config.yaml
WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /data/k8s/kube-config.yaml
coalesce.go:202: warning: destination for nodeSelector is a table. Ignoring non-table value []
coalesce.go:202: warning: destination for environment is a table. Ignoring non-table value []
coalesce.go:202: warning: destination for nodeSelector is a table. Ignoring non-table value []
coalesce.go:162: warning: skipped value for nodeSelector: Not a table.
coalesce.go:162: warning: skipped value for environment: Not a table.
coalesce.go:162: warning: skipped value for nodeSelector: Not a table.
Error: INSTALLATION FAILED: template: iste-icm-11/charts/icm/charts/icm-web/templates/wa-deployment.yaml:69:20: executing "iste-icm-11/charts/icm/charts/icm-web/templates/wa-deployment.yaml" at <eq .Values.persistence.type "nfs">: error calling eq: incompatible types for comparison
```

--- a/charts/icm-web/templates/wa-deployment.yaml
+++ b/charts/icm-web/templates/wa-deployment.yaml
@@ -66,7 +66,7 @@ spec:
         {{- else if eq .Values.persistence.pagecache.type "local" }}
         persistentVolumeClaim:
           claimName: "{{ template "icm-web.fullname" . }}-local-pagecache-pvc"
-        {{- else if eq .Values.persistence.type "nfs" }}
+        {{- else if eq .Values.persistence.pagecache.type "nfs" }}
         persistentVolumeClaim:
           claimName: "{{ template "icm-web.fullname" . }}-nfs-pagecache-pvc"
         {{- else if eq .Values.persistence.pagecache.type "cluster" }}

